testing/trino-product-tests/src/main/java/io/trino/tests/kafka/TestKafkaAvroWritesSmokeTest.java

```diff
@@ -58,6 +60,51 @@ public void testInsertPrimitiveDataType()
                         row("kasia", 9223372036854775807L, null, null));
     }
 
+    @Test(groups = {KAFKA, PROFILE_SPECIFIC_TESTS})
+    public void testInsertDefaultValue()
+    {
+        assertThat(query(format(
+                "INSERT INTO %s.%s (c_varchar) VALUES ('non-default')",
+                KAFKA_CATALOG,
+                DEFAULT_AVRO_VALUES_TABLE_NAME)))
+                .updatedRowsCountIsEqualTo(1);
+
+        assertThat(query(format(
+                "INSERT INTO %s.%s (c_bigint) VALUES (9223372036854775807)",
+                KAFKA_CATALOG,
+                DEFAULT_AVRO_VALUES_TABLE_NAME)))
+                .updatedRowsCountIsEqualTo(1);
+
+        assertThat(query(format(
+                "INSERT INTO %s.%s (c_double) VALUES (1234567890.123456789)",
+                KAFKA_CATALOG,
+                DEFAULT_AVRO_VALUES_TABLE_NAME)))
+                .updatedRowsCountIsEqualTo(1);
+
+        assertThat(query(format(
+                "INSERT INTO %s.%s (c_boolean) VALUES (false)",
+                KAFKA_CATALOG,
+                DEFAULT_AVRO_VALUES_TABLE_NAME)))
+                .updatedRowsCountIsEqualTo(1);
+
+        assertThat(query(format(
+                "INSERT INTO %s.%s VALUES (null, null, null, null)",
+                KAFKA_CATALOG,
+                ALL_DATATYPES_AVRO_TABLE_NAME)))
+                .updatedRowsCountIsEqualTo(1);
+
+        assertThat(query(format(
+                "SELECT * FROM %s.%s",
+                KAFKA_CATALOG,
+                DEFAULT_AVRO_VALUES_TABLE_NAME)))
+                .containsOnly(
+                        row("non-default", -9223372036854775808L, 3.14, true),
+                        row("default", 9223372036854775807L, 3.14, true),
+                        row("default", -9223372036854775808L, 1234567890.123456789, true),
+                        row("default", -9223372036854775808L, 3.14, false),
+                        row(null, null, null, null));
```

Comment:
I'd like one more column whose default value is `null`. It should behave the same as an explicit `null` being inserted for that column.